docs/csp-product.md

```diff
@@ -4,16 +4,17 @@ title: CSP products onboarding
 sidebar_label: Microsoft CSP
 ---
 
+Cloudesire can be used as cloud **Billing Engine** for Microsoft **Direct CSPs**
+to automate the selling of Office 365 licenses, Azure subscriptions and so on.
+
 This section describe how to onboard and start selling Microsoft product via its
-[Cloud Solution
-Provider](https://partner.microsoft.com/en-US/cloud-solution-provider/) (CSP)
-program on the marketplace.
-
-Currently the platform support only one CSP account per marketplace, so a
-registered software vendor cannot import and use their own Partner Center
-credentials. The platform administrator configure the credentials, import CSP
-products into the marketplace catalog and assign them to a particular software
-vendor account.
+[Cloud Solution Provider](https://partner.microsoft.com/en-US/cloud-solution-provider/)
+(CSP) program on the marketplace.
+
+The platform supports one **CSP account** per marketplace, so a registered
```

Comment:
this sounds like a deliberate limit ("so a vendor..cannot"), maybe reuse that "currently"

something like "The platform supports one **CSP account** per marketplace (currently you cannot use your own..."